src/database-dummy.cpp

```diff
@@ -30,13 +30,16 @@ bool Database_Dummy::saveBlock(const v3s16 &pos, const std::string &data)
 	return true;
 }
 
-std::string Database_Dummy::loadBlock(const v3s16 &pos)
+void Database_Dummy::loadBlock(const v3s16 &pos, std::string *block)
```

Comment:
If this gets compiled by a c++11 compiler, won't move semantics avoid the copy anyway?